gsdk/api-gen/Cargo.toml

```diff
@@ -25,4 +25,4 @@ subxt-codegen.workspace = true
 # TODO: inherit from workspace
 #
 # use "2.0.15" because subxt-codegen-0.29.0 requires it.
-syn = { version = "2.0.15", features = ["full", "parsing"] }
+syn = { version = "2.0.28", features = ["full", "parsing"] }
```

Comment:
For some reason it works with 2.0.28

Reply:
It anyway depends on v 25 atm in lock file